dlt/extract/exceptions.py

```diff
@@ -259,3 +259,9 @@ def __init__(self, source_name: str, schema_name: str) -> None:
 class IncrementalUnboundError(DltResourceException):
     def __init__(self, cursor_path: str) -> None:
         super().__init__("", f"The incremental definition with cursor path {cursor_path} is used without being bound to the resource. This most often happens when you create dynamic resource from a generator function that uses incremental. See https://dlthub.com/docs/general-usage/incremental-loading#incremental-loading-with-last-value for an example.")
+
+
+class ValidationError(ValueError, DltException):
+    def __init__(self, original_exception: Exception) ->None:
+        self.original_exception = original_exception
+        super().__init__(f"Schema validation failed: {original_exception}")
```

Comment:
`Extracted data item could not be validated with %s` where %s is a validator class (ie`PydanticValidator`). Also please pass the model and the data item to this error so people can ew. catch it and print the item